.editorconfig

```diff
@@ -0,0 +1,28 @@
+root = true
+
+[*]
+end_of_line = LF
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.html]
+indent_size = 4
+
+[*.html.twig]
+indent_size = 4
+
+[*.json]
+indent_style = tab
```

Comment:
What, really?

Reply:
I think our .editorconfig has always been this way. Martian widgets etc. also have this. Do we want to change it?

Reply:
Well our package.json files use 2 spaces, so I don't think this value is really being respected. Regardless, should definitely be space IMO

Reply:
line 24 shows package.json has a different ruleset, for 2 spaces, not tabs. Same as previously. Which I always hated, too.

Reply:
yeah not sure where that started. we could update it from [.editorconfig](https://github.com/FoxSportsAustralia/fs-default-project-config/blob/master/resources/.editorconfig) and stop copying it around